### PR TITLE
Pruning unused dependancies to reduce vendorjs from 6+mb to 50kb

### PR DESCRIPTION
**Pull request by mosesokemwa**
- [x] I have read [Wikonnects contributing guidelines](https://github.com/tunapanda/wikonnect/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request passes all tests.

In an effort to reduce emberJs.... vendorJs file from +6mb to 5kb we need to first prune all unused packages and files

### CODE REVIEW

File: server/utils/routesUtils/userRouteUtils.js [hunk 1]
```diff
@@ -21,10 +21,10 @@ async function getProfileImage(id) {
       let image = 'data:image/(png|jpg);base64,' + encode(getImage.Body);
       return image;
     } else {
-      return 'uploads/images/profile-placeholder.gif';
+      return '/uploads/images/profile-placeholder.gif';
```
Comment (0xMurage) [inline server/utils/routesUtils/userRouteUtils.js:24]: the picture is missing on the server side public folder

File: frontend/package.json [hunk 2]
```diff
@@ -14,24 +14,21 @@
     "build": "ember build",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
-    "start": "ember serve --proxy https://app.wikonnect.org/",
-    "start:local": "ember serve  --proxy http://localhost:3000",
+    "start": "ember serve  --proxy http://localhost:3000",
```
Comment (0xMurage) [inline frontend/package.json:17]: I am cool with this if it will work on production

**Review by 0xMurage — CHANGES_REQUESTED** [review]
add the picture on server side so we don't have weird placeholder views on prod.

File: server/utils/routesUtils/userRouteUtils.js (hunk 1 above)
Comment (mosesokemwa) [reply server/utils/routesUtils/userRouteUtils.js:24]: On your localhost you need to add it to the ``wikonnect/server/uploads/images`` here is the [absolute url](https://app.wikonnect.org/uploads/images/profile-placeholder.gif) to the image

File: frontend/package.json (hunk 2 above)
Comment (mosesokemwa) [reply frontend/package.json:17]: It will work on prod, I just dont like proxies to the server

**Review by 0xMurage — APPROVED** [review]

**Pull request merged**